packages/paste-website/src/components/icons-list/types.ts

```diff
@@ -0,0 +1,21 @@
+import {GenericIconProps} from '@twilio-paste/icons/esm/types';
+
+export interface IconComponent {
+  [key: string]: React.FunctionComponent<GenericIconProps>;
+}
+
+export interface IconObject {
+  name: string;
+  esm: string;
+  cjs: string;
+  component: React.FunctionComponent<GenericIconProps>;
```

Comment:
Might be good to capitalize this to make it more clear it's a react component and easier to destruct and use

Reply:
Done